wp-parsely.php

```diff
@@ -154,3 +154,54 @@ function parsely_integrations(): Integrations {
 
 	return $parsely_integrations;
 }
+
+/**
+ * Helper function to determine if this plugin is activated "network-wide" for a Multisite instance.
+ *
+ * @return boolean
+ */
+function is_network_active(): bool {
+	return is_multisite() && is_plugin_active_for_network( _plugin_basename() );
+}
+
+/**
+ * Get the relative path to the plugin entry file for use in e.g. specifying the plugin on the plugins list admin page.
+ * This is a wrapper on plugin_basename which caches the calculated result to avoid repeating the underlying work.
+ *
+ * @return string The cached result of the plugin basename.
+ */
+function _plugin_basename(): string {
+	static $basename;
+
+	if ( isset( $basename ) ) {
+		return $basename;
+	}
+
+	$basename = plugin_basename( PARSELY_FILE );
+	return $basename;
+}
+
+/**
+ * When this plugin was activated, redirect to the most appropriate settings page.
+ * If the plugin was activated network-wide for a multisite, the destination is the sites list.
+ * Otherwise, this will redirect to the wp-parsely settings URL for the current site.
+ *
+ * @param string $plugin The "basename" of the activated plugin.
+ * @param bool   $network_wide Was the plugin was network activated.
+ * @return void
+ */
+function redirect_to_settings_or_sites_list_on_activate( $plugin, $network_wide ) {
```

Comment:
I'm not really sure about the redirect. There might be the case when a user is network activating multiple plugins and this is too intrusive.